src/Core/L10n/L10n.php

```diff
@@ -458,4 +402,19 @@ public function getPokeVerbs()
 
 		return $arr;
 	}
+
+	/**
+	 * Creates a new L10n instance based on the given langauge
+	 *
+	 * @param string $lang The new language
+	 *
+	 * @return static A new L10n instance
+	 * @throws \Exception
+	 */
+	public function withLang(string $lang)
+	{
+		$newL10n = clone $this;
```

Comment:
Can you please check if the requested language is different from the current before cloning/loading?